client/src/components/CategoriesScreen.tsx

```diff
@@ -9,32 +9,34 @@ import {
   View,
 } from 'react-native';
 import { StackParamList } from '../../App';
-import { Category, emptyCategory } from '../constants/defaultCategories';
+import { emptyCategory } from '../constants/emptyMocks';
+import { IUser } from '../constants/interfaces';
 import { MESSAGES } from '../constants/messages';
+import { getUser } from '../helpers/api';
 import { STYLES } from '../styles/styles';
 import ModalEditCategory from './ModalEditCategory';
 import ModalNewCategory from './ModalNewCategory';
 import Navbar from './Navbar';
 
 type CategoriesScreenProps = {
   params: NativeStackScreenProps<StackParamList, 'Categories'>;
-  categories: Category[];
-  setCategories: Function;
+  user: IUser;
+  setUser: React.Dispatch<React.SetStateAction<IUser>>;
 };
 
 const getColorStyle = (color: string) => {
   return {
-    color: color,
+    color,
```

Comment:
```const getColorStyle = (color: string) => ({color});```
it could be even shorter, but I'd rather not create this separate function, this could be used inline

```[styles.category, ({color: category.color})]``` seems a lot shorter